src/components/VideoCard/VideoCard.jsx

```diff
@@ -0,0 +1,37 @@
+import useStore from '../../hooks/useStore';
+import StyledVideoContainer from '../StyledVideoContainer';
+import StyledVideoFrame from '../StyledVideoFrame';
+import StyledVideoTitle from '../StyledVideoTitle';
+
+export default function VideoCard() {
+	const videos = useStore(state => state.videos);
+
+	function getYouTubeId(url) {
+		url = url.split(/(vi\/|v=|\/v\/|youtu\.be\/|\/embed\/|\/shorts\/)/);
+		return url[2] !== undefined ? url[2].split(/[^0-9a-z_-]/i)[0] : url[0];
+	}
+
+	return (
+		<>
+			{videos.map(video => {
+				const videoID = getYouTubeId(video.YouTubeLink);
+				return (
+					<StyledVideoContainer key={video.id}>
+						<StyledVideoTitle>{video.videoTitle}</StyledVideoTitle>
+						<StyledVideoFrame
+							key={videoID}
+							width="180"
+							height="315"
+							aria-label="YouTube video"
```

Comment:
I would probably pass the title of the video here instead of the text `YouTube video`. Now depending of the length of your list, every video has the accessible name of `YouTube video`. So in the end it is not really accessible anymore because the videos cannot be separated 😉